sympy/polys/rootconditions.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+from sympy.polys.domains import EXRAW
+from sympy.polys.densebasic import dup_convert
+from sympy.polys.densetools import dup_clear_denoms
+from sympy.external.gmpy import MPQ
+from sympy.core.mul import prod
+
+def routh_hurwitz(p: Poly):
+    """
+    Return the conditions for the polynomial to have all roots with negative
+    real part.
+
+    """
+    coeffs = p.rep.to_list()
+    domain = p.domain
+
+    conds = routh_hurwitz_dom(coeffs, domain)
+
+    # return And(*[domain.to_sympy(c) > 0 for c in conds])
+    return [domain.to_sympy(c) for c in conds]
+
+
+def routh_hurwitz_dom(p: list[Er], K: Domain[Er]) -> list[Er]:
```

Comment:
The general convention in polys is that this function would be called `dup_routh_hurwitz`. See e.g. here and look at surrounding code:
https://github.com/sympy/sympy/blob/0147754bcf0d3b5c0d07c283803e9ff69a9d9454/sympy/polys/densebasic.py#L577